packages/ui/src/components/organisms/RegionModal/styles.scss

```diff
@@ -14,7 +14,7 @@
   // --------------------------------------------------------
   // Structural Styles
   // --------------------------------------------------------
-  [data-fs-region-modal-input] {
```

Comment:
Since we are not refering to this data-attr anywhere else but RegionModal, it won't break 👍 

Do you think we should remove only this token from the Design Tokens table [here](https://developers.vtex.com/docs/guides/faststore/organisms-region-modal)?

Reply:
Why should we remove it?

Reply:
just because we're not using it. But I think it's better to keep it, perhaps somebody is already using it.